tools/preprocess_data.py

```diff
@@ -72,6 +72,18 @@ def get_args():
         "--glob-pattern", type=str, default=None, help="A glob pattern to filter files to read. Default: None"
     )
 
+    p3 = sp.add_parser(name="parquet")
+    p3.add_argument(
+        "--dataset",
+        type=str,
+        required=True,
+        help="Path to a .paquet file or hf:// path",
```

Comment:
```suggestion
        help="Path to a .parquet file or a folder containing multiple .parquet files",
```
Let's add the same comment as with the `JsonlReader`.